adoc/MAIN.release-notes.adoc

```diff
@@ -57,36 +57,44 @@ skuba should disable firewalld when bootstrapping/joining a node. Adds a startup
 
 FILLIN
 
-=== Known issues
+[[known-issues-420]]
+=== Known Issues
 
-==== Updating packages without updating k8s
+==== `skuba-update` Error: `patterns-caasp-Node` Conflicts with {crio} Update
 
-FILLIN
+If your cluster is not up-to-date, meaning it is not in the latest {kube} version, `skuba-update` will try to install the latest version of {crio}, which will create a conflict with
+the currently installed {kube} packages.
+
+More precisely, you might encounter an error similar to this:
+
+"patterns-caasp-Node conflicts with {crio}"
+
+In that case, the recommended solution is to upgrade the cluster by running `skuba node upgrade` on each node.
```

Comment:
what about:

"In that case, the recommended solution is to upgrade the cluster to the latest Kubernetes version available, this can be done by running the regular Kubernetes upgrade procedure of CaaSP [link to kubernetes upgrade docs] based on the command `skuba node upgrade`"

Reply:
Ah thanks, yes, that makes it better understandable I think.